unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/Helpers/BIWTestHelper.cs

```diff
@@ -192,4 +194,16 @@ public static void CreateNFT()
             BIWNFTController.i.NftsFeteched(owner);
         }
     }
+
+    public static IEnumerator CreateShapeForEntity(ParcelScene scene, IDCLEntity entity)
+    {
+        TestHelpers.CreateAndSetShape(scene, entity.entityId, DCL.Models.CLASS_ID.GLTF_SHAPE, JsonConvert.SerializeObject(
+            new
+            {
+                src = TestAssetsUtils.GetPath() + "/GLB/Trunk/Trunk.glb"
+            }));
+
+        LoadWrapper gltfShape = GLTFShape.GetLoaderForEntity(scene.entities[entity.entityId]);
+        yield return new DCL.WaitUntil(() => gltfShape.alreadyLoaded);
+    }
```

Comment:
We already have `TestHelpers.CreateEntityWithGLTFShape`. This implementation is absolutely needed?.

Reply:
The function that you mention also creates the entity and returns the GLTFComponent. I need the entity to configure the BIWEntity, trying to look the entity in the GLTFComponent looks like a bit hacky to me

Reply:
I'd prefer to have a single way of doing things if possible. I think we are creating clutter here, if you really feel that this is needed, we can move this to `TestHelpers`?